### ISSUE TITLE
Cannot retreive FileRef property from getItemsByCAMLQuery() method.

### ISSUE THREAD

**Issue by drewcook** [pnp/pnpjs#135]
#### Category

[ ] Enhancement

[x ] Bug

[ ] Question

#### Version
v1.1.0 - I am using the segmented pnp/sp library

#### Expected / Desired Behavior / Question

To have the FileRef property returned from a CAML query GET.

#### Observed Behavior

Does not return FileRef property.

#### Steps to Reproduce

`import { sp } from "@pnp/sp";`

The following will return an array of the items, but each one does not have the specified FileRef property:
```
const viewFields = 
	"<ViewFields>
		<FieldRef Name='Alt_x0020_Text' />
		<FieldRef Name='AlternateThumbnailUrl' />
		<FieldRef Name='FileRef' />
		<FieldRef Name='ID' />
		<FieldRef Name='Title' />>
	</ViewFields>";
const queryOptions = "<QueryOptions><ViewAttributes Scope='RecursiveAll'/></QueryOptions>";

sp.web.lists.getByTitle("Media").getItemsByCAMLQuery({
	ViewXml: "<View>"+viewFields+camlQuery+queryOptions+"</View>"
}).then(media => { console.log(media); });
```
This should return a FileRef property just like a regular get() using select() would:

`sp.web.lists.getByTitle("Media").items.select("Alt_x0020_Text", "AlternateThumbnailUrl", "FileRef", "ID", "Title").get().then(media => { console.log(media); });`

The latter returns the FileRef with no issues.  The former does not.

**Comment by koltyakov** [comment]
Hey @drewcook,

There are some typos in the provided code snippet, like `>>` or using `"` as if it were template string. Yet I suppose that it's just an artifact of copying and pasting to the issue body.

`getItemsByCAMLQuery` method suppose `expands`. In case of `FileRef` it should be passed to the expands. Check the sample below:

```typescript
import { sp } from '@pnp/sp';

const viewFields =  `
    <ViewFields>
        <FieldRef Name='Alt_x0020_Text' />
        <FieldRef Name='AlternateThumbnailUrl' />
        <FieldRef Name='FileRef' />
        <FieldRef Name='ID' />
        <FieldRef Name='Title' />
    </ViewFields>
`;

const queryOptions = `<QueryOptions><ViewAttributes Scope='RecursiveAll'/></QueryOptions>`;
const camlQuery = '';

sp.web.lists.getByTitle('Documents')
    .getItemsByCAMLQuery({
        ViewXml: `<View>${viewFields}${camlQuery}${queryOptions}</View>`
    }, 'FileRef')
    .then(console.log)
    .catch(console.log);
```

![image](https://user-images.githubusercontent.com/7816483/41496521-7978afea-714a-11e8-95e1-2fb727f09b34.png)

**Comment by drewcook** [comment]
@koltyakov Thank you, that is definitely working for me!  I did not see any detailed documentation on the method, as it is kind of lacking.  I did not know that there was a second optional parameter that could be passed in that would represent expands.  Furthermore, there was no documentation stating that FileRef in particular (or any others for that matter like ContentType, etc) would need to be expanded when using the method.

The documentation is lacking for a good portion of the library.  We love using it on our team but it could be better.

Thank you for the help!
